pom.xml

```diff
@@ -73,6 +73,10 @@
             <name>George Gastaldi</name>
             <email>gegastaldi@gmail.com</email>
         </contributor>
+        <contributor>
+            <name>Damir Alibegovic</name>
+            <email>alibegovic.damir@gmail.com</email>
+        </contributor>
```

Comment:
We don't really maintain this one anymore, I should probably remove the one that is there but you should add your name here: https://github.com/hibernate/hibernate-validator/blob/master/copyright.txt#L13